discovery/discovery.go

```diff
@@ -36,8 +36,13 @@ func NewOrchestratorPool(bcast common.Broadcaster, uris []*url.URL, score float3
 		glog.Error("Orchestrator pool does not have any URIs")
 	}
 	infos := make([]common.OrchestratorLocalInfo, 0, len(uris))
-	for _, uri := range uris {
-		infos = append(infos, common.OrchestratorLocalInfo{URL: uri, Score: score})
+	if len(uris) == 2 {
```

Comment:
Could you add a comment here about why the behaviour is different with 2 URIs and how we know the first is trusted and the second untrusted?

Reply:
Good catch, it's a debugging piece I forgot.